gengokit/template/NAME-service/NAME-cli-client/client_submain.gotemplate

```diff
@@ -0,0 +1,126 @@
+package main
```

Comment:
That's a good point; it ended up a separate file during coding, but it definitely makes much more sense for them to live together within the same file.